tests/unit/lms/resources/_js_config/__init___test.py

```diff
@@ -146,11 +144,9 @@ def test_it(self, bearer_token_schema, context, grant_token_service, js_config):
             "rpcServer": {"allowedOrigins": ["http://localhost:5000"]},
         }
 
-    def test_it_raises_if_theres_no_ApplicationInstance(
-        self, application_instance_service, js_config
-    ):
-        application_instance_service.get_current.side_effect = (
-            ApplicationInstanceNotFound
+    def test_it_raises_if_theres_no_ApplicationInstance(self, context, js_config):
+        type(context).application_instance = mock.PropertyMock(
```

Comment:
https://docs.python.org/3/library/unittest.mock.html#unittest.mock.PropertyMock

```Because of the way mock attributes are stored you can’t directly attach a PropertyMock to a mock object. Instead you can attach it to the mock type object:

```